### ISSUE TITLE
use instruments to check leaks , found memory leak.

### ISSUE THREAD

**Issue by dahuangfengda** [Alamofire/Alamofire#1357]
it shows this code leaks : `let JSON = try NSJSONSerialization.JSONObjectWithData(validData, options: options)`
the method is     ```
public static func JSONResponseSerializer(
        options options: NSJSONReadingOptions = .AllowFragments)
        -> ResponseSerializer<AnyObject, NSError>
    {
        return ResponseSerializer { _, _, data, error in
            guard error == nil else { return .Failure(error!) }
            
            guard let validData = data where validData.length > 0 else {
                let failureReason = "JSON could not be serialized. Input data was nil or zero length."
                let error = Error.errorWithCode(.JSONSerializationFailed, failureReason: failureReason)
                return .Failure(error)
            }
            
            do {
                let JSON = try NSJSONSerialization.JSONObjectWithData(validData, options: options)
                return .Success(JSON)
            } catch {
                return .Failure(error as NSError)
            }
        }
    }
```

**Comment by cnoon** [comment]
Hi @dahuangfengda, without any sample code to go on here, we can't reproduce the leak you're seeing. I'm going to close this issue out for now since there's not enough information to investigate. If you end up posting a sample project that can reproduce the leak, we'll gladly re-open and investigate.

Cheers. 🍻

**Comment by AntiMoron** [comment]
@cnoon  
Just the example project. 

![image](https://cloud.githubusercontent.com/assets/6587734/17389407/9294c612-5a36-11e6-8107-caa1494f9ed9.png)


Yes, there are leaks occured on sending network requests.

BTW, when running objective-c sample project of AFNetworking, no leaks at all.
But I've tried to use AFNetworking in my swift project through bridging header, leaks on every network request.

Through searching I found this blog:
http://footle.org/2015/10/10/fixing-a-swift-memory-leak/
